<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="DownloadQueueView" name="downloadQueueView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionSpeedLimit"/>
    <addaction name="separator"/>
    <addaction name="actionForceStart"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuOther1">
     <property name="title">
      <string>Other</string>
     </property>
     <addaction name="actionRemoveAll"/>
     <addaction name="separator"/>
     <addaction name="actionRemoveWaiting"/>
    </widget>
    <widget class="QMenu" name="menuOther2">
     <property name="title">
      <string>Other</string>
     </property>
     <addaction name="actionRemoveSelected"/>
     <addaction name="separator"/>
     <addaction name="actionRemoveFailed"/>
     <addaction name="actionRemovePaused"/>
    </widget>
    <addaction name="actionRemoveCompleted"/>
    <addaction name="menuOther1"/>
    <addaction name="separator"/>
    <addaction name="menuOther2"/>
   </widget>
   <widget class="QMenu" name="menuDownload">
    <property name="title">
     <string>Download</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="separator"/>
    <addaction name="actionResume"/>
    <addaction name="actionPause"/>
    <addaction name="actionCancel"/>
    <addaction name="separator"/>
    <addaction name="actionTop"/>
    <addaction name="actionUp"/>
    <addaction name="actionDown"/>
    <addaction name="actionBottom"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSelectNone"/>
    <addaction name="actionInvertSelection"/>
    <addaction name="actionSelectCompleted"/>
    <addaction name="separator"/>
    <addaction name="actionOneMoreSegment"/>
    <addaction name="actionOneFewerSegment"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuDownload"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QToolBar" name="downloadBar">
   <property name="windowTitle">
    <string>View toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="separator"/>
   <addaction name="actionResume"/>
   <addaction name="actionPause"/>
   <addaction name="actionCancel"/>
   <addaction name="separator"/>
   <addaction name="actionTop"/>
   <addaction name="actionUp"/>
   <addaction name="actionDown"/>
   <addaction name="actionBottom"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/default/scalable/actions//add-batch.svg</normaloff>:/resources/icons/default/scalable/actions//add-batch.svg</iconset>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/play-stop.svg</normaloff>:/resources/icons/default/scalable/actions/play-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/play-pause.svg</normaloff>:/resources/icons/default/scalable/actions/play-pause.svg</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/move-up.svg</normaloff>:/resources/icons/default/scalable/actions/move-up.svg</iconset>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
   <property name="shortcut">
    <string>Alt+PgUp</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/move-top.svg</normaloff>:/resources/icons/default/scalable/actions/move-top.svg</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
   <property name="shortcut">
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/move-down.svg</normaloff>:/resources/icons/default/scalable/actions/move-down.svg</iconset>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
   <property name="shortcut">
    <string>Alt+PgDown</string>
   </property>
  </action>
  <action name="actionBottom">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/move-bottom.svg</normaloff>:/resources/icons/default/scalable/actions/move-bottom.svg</iconset>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
   <property name="shortcut">
    <string>Alt+End</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/play-resume.svg</normaloff>:/resources/icons/default/scalable/actions/play-resume.svg</iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/select-all.svg</normaloff>:/resources/icons/default/scalable/actions/select-all.svg</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionInvertSelection">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/select-invert.svg</normaloff>:/resources/icons/default/scalable/actions/select-invert.svg</iconset>
   </property>
   <property name="text">
    <string>Invert Selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionOneMoreSegment">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/segment-add.svg</normaloff>:/resources/icons/default/scalable/actions/segment-add.svg</iconset>
   </property>
   <property name="text">
    <string>One More Segment</string>
   </property>
  </action>
  <action name="actionOneFewerSegment">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/segment-remove.svg</normaloff>:/resources/icons/default/scalable/actions/segment-remove.svg</iconset>
   </property>
   <property name="text">
    <string>One Fewer Segment</string>
   </property>
  </action>
  <action name="actionForceStart">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/default/scalable/actions/play-resume-force.svg</normaloff>:/resources/icons/default/scalable/actions/play-resume-force.svg</iconset>
   </property>
   <property name="text">
    <string>Force Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionRemoveCompleted">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/remove-completed.svg</normaloff>:/resources/icons/default/scalable/actions/remove-completed.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Completed</string>
   </property>
  </action>
  <action name="actionRemoveWaiting">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/remove-waiting.svg</normaloff>:/resources/icons/default/scalable/actions/remove-waiting.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Waiting</string>
   </property>
   <property name="toolTip">
    <string>Remove Waiting</string>
   </property>
   <property name="shortcut">
    <string>Alt+Del</string>
   </property>
  </action>
  <action name="actionSpeedLimit">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/default/scalable/actions/limit-speed.svg</normaloff>:/resources/icons/default/scalable/actions/limit-speed.svg</iconset>
   </property>
   <property name="text">
    <string>Speed Limit...</string>
   </property>
   <property name="toolTip">
    <string>Speed Limit...</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/select-none.svg</normaloff>:/resources/icons/default/scalable/actions/select-none.svg</iconset>
   </property>
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="toolTip">
    <string>Select None</string>
   </property>
  </action>
  <action name="actionSelectCompleted">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/select-completed.svg</normaloff>:/resources/icons/default/scalable/actions/select-completed.svg</iconset>
   </property>
   <property name="text">
    <string>Select Completed</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/remove-all.svg</normaloff>:/resources/icons/default/scalable/actions/remove-all.svg</iconset>
   </property>
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionRemoveSelected">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/remove-downloaded.svg</normaloff>:/resources/icons/default/scalable/actions/remove-downloaded.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Selected</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRemoveFailed">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/remove-stopped.svg</normaloff>:/resources/icons/default/scalable/actions/remove-stopped.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Failed</string>
   </property>
  </action>
  <action name="actionRemovePaused">
   <property name="icon">
    <iconset resource="../../../src/resources.qrc">
     <normaloff>:/resources/icons/default/scalable/actions/play-pause.svg</normaloff>:/resources/icons/default/scalable/actions/play-pause.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Paused</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DownloadQueueView</class>
   <extends>QWidget</extends>
   <header>Widgets/DownloadQueueView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../src/resources.qrc"/>
 </resources>
 <connections/>
</ui>
